fe/fe-core/src/main/java/com/starrocks/analysis/ColumnDef.java

```diff
@@ -59,22 +58,36 @@
      */
     public static class DefaultValue {
```

Comment:
I think it is better to create a new struct to represent DefaultValue.
This struct is used to describe syntax object. And Column's DefaultValue is a parsed object.